sys-fs/compsize/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>adebeus@gmail.com</email>
+  </maintainer>
+  <maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
+  </maintainer>
+  <longdescription>utility to measure btrfs compression ratio</longdescription>
```

Comment:
Remove this, please. It's only necessary when you have a very long description that doesn't fit in ebuild.